pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -167,9 +167,24 @@ type CertificateSpec struct {
 	// If unset, this defaults to 1/3 of the issued certificate's lifetime.
```

Comment:
I think that this comment should be moved to `RenewBeforePercentage`. It now makes more sense to say that `RenewBeforePercentage` defaults to 33%.

Reply:
The reason I didn't make this change is that 1/3 is not exactly the same as 33%, and implementation wise I was trying to keep things a no-op for anyone who hasn't opted into using `RenewBeforePercentage`.